tests/Unit/EventTest.php

```diff
@@ -154,6 +154,45 @@ public function test_cron_life_time(): void
         );
     }
 
+    public function test_get_from(): void
```

Comment:
What about "use data provider (can be one for both cases) to test `string` and `\DateTime` in from/to"?

Reply:
In progress.